css/style.css

```diff
@@ -379,44 +652,251 @@
   .data__speed-block {
     padding-right: 55px;
   }
-}
 
-@media (min-width: 1024px) and (max-width: 1199px) {
-  .menubar__link:nth-child(5) {
-    margin-right: 120px;
+  .data__figures {
+    font-weight: 600;
+  }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1200px;
+  }
+
+  .wrap-information {
+    height: 863px;
+  }
+
+  .red-first {
+    height: 100%;
+  }
+
+  .drive {
+    font-size: 15px;
+  }
+
+  .description__bag {
+    padding-bottom: 150px;
+  }
+
+  .wrap__img {
+    width: 56%;
+    height: 145px;
+    padding-left: 529px;
+    top: 138px;
+  }
+
+  .wrap__img-car {
+    width: 72%;
+    right: -349px;
+    top: 35px;
+  }
+
+  .wrap__whs-img {
+    bottom: 0;
+    right: 0;
+  }
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 120px;
+  }
+
+  .main-tesla {
+    display: flex;
+    justify-content: center;
+  }
+
+  .tesla-bgd {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+    top: -75px;
+    left: 0;
+  }
+
+  .tesla-car {
+    padding: 65px 0 20px 4px;
+  }
+
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 22px;
+  }
+
+  .data__speed-block {
+    padding-right: 35px;
+  }
+
+  .data__figures {
+    font-weight: 600;
+  }
+
+  .test-btn {
+    padding-right: 0;
+  }
+
+  .testdrive {
+    margin-right: 60px;
+  }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+
+  .wrap-information, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)